packages/story-editor/src/components/library/panes/pageTemplates/savedPageTemplate.js

```diff
@@ -244,43 +300,52 @@ function SavedPageTemplate(
           </ElementsWrapper>
         )}
         {isActive && <TemplateInsertionOverlay showIcon={false} />}
-        <ActionButton
-          ref={insertButtonRef}
-          onClick={(e) => {
-            e.stopPropagation();
-            onClick(e);
-          }}
-          aria-label={__('Use template', 'web-stories')}
-          $display={isActive}
-          tabIndex={index === 0 ? 0 : -1}
-        >
-          <Icons.PlusFilledSmall />
-        </ActionButton>
-        <DeleteButton
-          ref={deleteButtonRef}
-          $display={isActive}
-          onClick={(e) => {
-            e.stopPropagation();
-            handleDelete(page, e);
-          }}
-          aria-label={__('Delete Page Template', 'web-stories')}
-          tabIndex={isActive ? 0 : -1}
-        >
-          <Icons.TrashFilledSmall />
-        </DeleteButton>
+        {page.title && (
+          <TemplateTitleContainer isActive={isActive}>
+            <Text.Span size={TextSize.Small}>
+              {title !== '' ? title : 'Untitled'}
```

Comment:
Ditto here about translation.

```suggestion
              {title !== '' ? title : __('Untitled', 'web-stories')}
```